### ISSUE TITLE
Change "NA" to "None" or 0

### ISSUE THREAD

**Issue by julienlim** [Tendrl/ui#658]
For views where "NA" is used, NA is too vague.  Please instead use the following:

Hosts List:
* For NA (for Cluster), change to None (if no Cluster)

Hosts List, Volumes List:
* For NA (for Alerts), change to 0 when there are no Alerts

![screen shot 2017-10-03 at 1 56 09 pm](https://user-images.githubusercontent.com/5589919/31140592-d790de12-a842-11e7-85e9-c996bbdf13c4.png)

@gnehapk @a2batic @Tendrl/tendrl-qe @mcarrano

**Comment by mbukatov** [comment]
I suggest to use zero value only when the value from backend is really reported as zero, and NA when the value is not available (so that there is nothing to report).

@julienlim Would it be ok to use `NA` for cases when the value is not available at all? I would be ok with that.

Which means that in this case, we need to figure out where the problem actually is:

* value is not reported by backed at all
* ui doesn't ask for the value and then the ui reports the value as not available
* ui receives the value, but represents it in a wrong way

In other words, this is not just a sheer change of the label, and we need to be sure that the backed works before trying to fix this.

**Comment by mcarrano** [comment]
I think there are actually three conditions that should be differentiated. 

* A "0" value. Agree that in this case we should display 0 as in "0 alerts".
* A null association, e.g. where a host has no associated cluster. In this case I think displaying "None" makes sense.
* There is no data available. I asked around a bit with the UXD team on this one. While we have no definite standard, I recommend "No Data." It's more specific than "NA" but shorter than "Data Unavailable" or "Not Available"

**Comment by cloudbehl** [comment]
@julienlim The Issue has been fixed, Please verify with the master build(tendrl-ui-1.5.2-20171011T095120.b554d56.src.rpm).

**Comment by julienlim** [comment]
I've confirmed that when there are no alerts, it says No Data.
![screen shot 2017-10-24 at 2 05 21 pm](https://user-images.githubusercontent.com/5589919/32003675-d0d833e0-b96d-11e7-99ca-39e03daec08a.png)

Have still to verify when no cluster present.

**Comment by julienlim** [comment]
Confirmed that Cluster column says "None" when node is not part of a Gluster cluster.
<img width="1209" alt="screen shot 2017-10-25 at 2 02 56 pm" src="https://user-images.githubusercontent.com/5589919/32014866-549026d8-b98d-11e7-9afb-5fbb5123e492.png">
